tasks/main.yml

```diff
@@ -78,61 +87,61 @@
 # be mentioned explicitly in the output of the command "systemctl
 # status mongod.service".
 - name: Test for AssertPathIsMountPoint lines
-  command: grep -c ^AssertPathIsMountPoint= /lib/systemd/system/mongod.service
-  register: test_assertpathismountpoint
-  ignore_errors: yes
+  ansible.builtin.command: grep -c ^AssertPathIsMountPoint= /lib/systemd/system/mongod.service
   changed_when: false
+  ignore_errors: yes
+  register: test_assertpathismountpoint
 - name: Make sure mongo data file system is mounted before starting
```

Comment:
Should these three tasks that all have a `when: test_assertpathismountpoint.rc != 0` be moved into a `block`?

Reply:
Please see commit 491a4a1.